### DISCUSSION TITLE
Table of contents and links to heading anchor tag incorrectly positioning

### DISCUSSION THREAD

**Discussion by mdgrant** [quarto-dev/quarto-cli#8082] [Q&A]
### Description

When clicking on a table of contents item in web pages used to report results of an evidence synthesis, a second click is required for it to appear at top of the screen (here is one page to try  https://mdgrant.github.io/geriatrics_synth/kq3.html). Similarly, links to heading anchors from other pages do not land positioning the header correctly. For example, the links to tables and figures from https://mdgrant.github.io/geriatrics_synth/soe_gt.html. But once there, clicking on the table of contents will bring it to the correct position. I assume both are related.

**Comment by mcanouil** [comment]
Could you not hide links behind words that do not tell/explain what the links are?
This is an accessibility request.
"This" and "page" are not meaningful nor specific to your links.

**Reply by mdgrant** [reply]
Apologies. I edited the description.

**Reply by mcanouil** [reply]
No worries. We all have to learn to be more accessible (including myself).
Thanks for editing.

**Comment by mcanouil** [comment]
This is the results of JavaScript and CSS, since this affects websites but not simple HTML document, there might be something to be done in the Bootstrap (Anchor.js) configuration.
CC @dragonstyle 

Note that, it's not the table of content.
Clicking directly on anchored links will produce the same results.
